bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqHandler.java

```diff
@@ -0,0 +1,986 @@
+                    }
+                    break;
+                case "roomType":
+                    newobj.addProperty("RoomType", command.toString());
+                    changeSettings(newobj);
+                    break;
+                case "logLevel":
+                    String ll = command.toString();
+                    if (ll.equals("Error") || ll.equals("Warning") || ll.equals("Info")) {
+                        newobj.addProperty("Logging", ll);
+                        changeSettings(newobj);
+                    } else {
+                        logger.error(
+                                "air-Q - airqHandler - handleCommand(): {} should be set to {} but it isn't a correct setting for the power frequency suppression (only 50Hz or 60Hz)",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                case "averagingRhythm":
+                    try {
+                        newobj.addProperty("SecondsMeasurementDelay", Integer.parseUnsignedInt(command.toString()));
+                    } catch (Exception exc) {
+                        logger.error(
+                                "air-Q - airqHandler - handleCommand(): {} only accepts an integer value, and {} is not.",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                case "powerFreqSuppression":
+                    String newFreq = command.toString();
+                    if (newFreq.equals("50Hz") || newFreq.equals("60Hz") || newFreq.equals("50Hz+60Hz")) {
+                        newobj.addProperty("Rejection", newFreq);
+                        changeSettings(newobj);
+                    } else {
+                        logger.error(
+                                "air-Q - airqHandler - handleCommand(): {} should be set to {} but it isn't a correct setting for the power frequency suppression (only 50Hz or 60Hz)",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                default:
+                    logger.error(
+                            "air-Q - airqHandler - handleCommand(): unknown command {} received (channelUID={}, value={})",
+                            command, channelUID, command);
+            }
+        }
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("air-Q - airqHandler - initialize(): ipaddress={}",
+                getThing().getConfiguration().get("ipAddress"));
+        updateStatus(ThingStatus.UNKNOWN);
+        if (getThing().getConfiguration().get("ipAddress") != null) {
+            ipaddress = getThing().getConfiguration().get("ipAddress").toString();
+        }
+        if (getThing().getConfiguration().get("password") != null) {
+            password = getThing().getConfiguration().get("password").toString();
+        }
```

Comment:
```suggestion
AirqConfiguration config = new AirqConfiguration(); //declare this globally at top of class.
//Lines below use here.
config = getConfig().as(AirqConfiguration.class);
            password = config.password; 
            ipaddress = config.ipaddress;
```

You don't need to use a local password and ipaddress variables as you can just use config.password and config.ipaddress instead anywhere you wish. Also it should be camelCase for ipAddress.